python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -2058,6 +2058,65 @@ def testBackground( self ) :
 		r.option( "ai:background", None )
 		self.assertEqual( arnold.AiNodeGetPtr( options, "background" ), None )
 
+	def testVDBs( self ) :
+
+		import IECoreVDB
+
+		tmpFile = self.temporaryDirectory() + "/test.ass"
+
+		r = GafferScene.Private.IECoreScenePreview.Renderer.create(
+			"Arnold",
+			GafferScene.Private.IECoreScenePreview.Renderer.RenderType.SceneDescription,
+			tmpFile
+		)
+
+		attributes = IECore.CompoundObject( {
+			"ai:volume:velocity_scale" : IECore.FloatData( 10 ),
+			"ai:volume:velocity_fps" : IECore.FloatData( 25 ),
+			"ai:volume:velocity_outlier_threshold" : IECore.FloatData( 0.5 ) } )
+
+		# Camera needs to be added first as it's being used for translating the VDB.
+		# We are doing the same when translating actual Gaffer scenes here:
+		# https://github.com/GafferHQ/gaffer/blob/master/src/GafferScene/Render.cpp#L254
+		r.camera(
+			"testCamera",
+			IECoreScene.Camera(
+				parameters = {
+					"shutter_start" : 10.75,
```

Comment:
This isn't how shutters are [meant to be specified](https://github.com/GafferHQ/gaffer/blob/master/include/GafferScene/Private/IECoreScenePreview/Renderer.h#L238); we're just getting lucky because the IECoreArnold camera converter copies over any parameters it doesn't know about. See the GafferDelight tests for an [example of the right way](https://github.com/GafferHQ/gaffer/blob/master/python/GafferDelightTest/IECoreDelightPreviewTest/RendererTest.py#L413).